src/components/order-service/order-service-details.html

```diff
@@ -1,16 +1,3 @@
-<div ng-if="$ctrl.versions" uib-dropdown class="input-group-btn version-dropdown">
-  <button uib-dropdown-toggle type="button" class="btn btn-default" uib-tooltip="Select Version" tooltip-placement="top">
-    {{$ctrl.selectedVersion}}
-    <span class="caret"></span>
-  </button>
-  <ul uib-dropdown-menu>
-    <li ng-repeat="version in $ctrl.versions">
-      <a role="menuitem" tabindex="-1" ng-click="$ctrl.selectedVersion = version">
-        {{version}}
-      </a>
-    </li>
-  </ul>
-</div>
 <div ng-bind-html="$ctrl.description | linky : '_blank'"></div>
 <!-- TODO: truncate long text -->
```

Comment:
We'd have to remove the `linky` call if we use `truncate-long-text` since it might truncate in the middle of a link :/

Wondering if a scrollbar here is sufficient for very long descriptions.